### PR TITLE
[Term entry] Pillow library method: .Transpose()

### LINKED ISSUE #4625: [Term Entry] Python pillow .transpose()
### Reason/inspiration (optional)

We would like a new term entry on the '.transpose()' method for the `Pillow` library in Python. The entry should go in a new file under `docs/content/pillow/concepts/image/terms/transpose/transpose.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by DevIggor**
Description
-Added the .Transpose() term for Pillow
-Closes Issue #4625

Issue Solved
Closes #4625

Type of Change
-Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by DevIggor** [comment]
@cigar-galaxy82 Hi 👋 Please, could you review my code on the new PR? the old one (https://github.com/Codecademy/docs/pull/4752) was closed by orientation of @letsbuildanything due to issues with the CLAassistant.

**Comment by letsbuildanything** [comment]
Thank you so much @DevIggor, for resubmitting this PR and for your patience with the CLA issue. Your effort and dedication are greatly appreciated, and we are grateful for your contribution to the Codecademy Docs! 🙌

@cigar-galaxy82 Please proceed with your first review as soon as possible.

**Comment by DevIggor** [comment]
review maded

**Review by cigar-galaxy82 — APPROVED** [review]
Hi, all the changes have been made as required

**Comment by cigar-galaxy82** [comment]
Hi @DevIggor as suggested earlier all the changes have been made so now second review can be done

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/transpose/transpose.md:16]:
```suggestion
---
Title: '.transpose()'
Description: 'Flips or rotates an image according to the specified operation.'
Subjects:
  - 'Code foundations'
  - 'Computer science'
Tags:
  - 'Methods'
  - 'Pillow'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
  - 'paths/data-science'
  - 'paths/data-science-foundations'
```

Comment (mamtawardhani) [reply content/pillow/concepts/image/terms/transpose/transpose.md:16]: The description should always start with a verb, and two tags are enough.

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/transpose/transpose.md:19]:
```suggestion
In Pillow, The **`.transpose()`** method is used to flip or rotate an image in 90-degree steps.
```

Comment (mamtawardhani) [reply content/pillow/concepts/image/terms/transpose/transpose.md:19]: An inline code needs to be inclosed within backticks always (` code `)

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/transpose/transpose.md:33]: Keep these out of the code block since all of them are not a part of the generic syntax. Also, just below the syntax, we should use a format:

- `Image`: Describe this parameter
- `method`: Describe this parameter and say it has the following types:
  - `FLIP_LEFT_RIGHT`: Flips the image horizontally.
  - and so on
 
So this way you can cover the parameter descriptions.

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/transpose/transpose.md:39]: Keep this outside the code block.

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/transpose/transpose.md:49]: let us not save the image, rather use the `.show()` to show both the images, the original one and the flipped one:

```
import PIL
from PIL import Image

# Read the image
im = Image.open("bird-thumbnail.jpg")
im.show()

# Flip image
out = im.transpose(PIL.Image.FLIP_LEFT_RIGHT)
out.show()
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @DevIggor, the entry is nicely written!

However two things:
1. Keep both the images - the original and the flipped image inside the "media" folder in docs, not in the pillow folder, please delete that.
2. I've suggested some changes for this entry, please modify those at your earliest convenience, feel free to dicuss any concerns. Thank you! 😄

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @DevIggor the entry is ready to be merged but we just require one change from your side, please remove the image of the visual studio code from this branch. Only keep the two output images. Thank you! 😄

**Comment by mamtawardhani** [comment]
Hey @DevIggor! Please remove the `content/pillow/concepts/image/terms/transpose/image.png` from this PR so that we can merge this at the earliest. Thank you! 😃

**Comment by DevIggor** [comment]
> Hey @DevIggor! Please remove the `content/pillow/concepts/image/terms/transpose/image.png` from this PR so that we can merge this at the earliest. Thank you! 😃

Sorry for that, I don't know it ended up there 😅

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the changes @DevIggor! The entry looks great! 😃 

It is ready to be merged! 🚀

**Pull request merged**
